README.md

```diff
@@ -46,5 +49,51 @@ $ rekcurd startapp {Your application name}
 $ python -m unittest
 ```
 
+
 ## Kubernetes support
-Rekcurd can be run on Kubernetes. See [here](https://github.com/rekcurd/community).
+Rekcurd can be run on Kubernetes. See [community repository](https://github.com/rekcurd/community).
+
+
+## Type definition
+### `PredictLabel` type
+*V* is the length of feature vector.
+
+|Field |Type |Description |
+|:---|:---|:---|
+|input <BR>(required) |One of below<BR>- string<BR>- bytes<BR>- string[*V*]<BR>- int[*V*]<BR>- double[*V*] |Input data for inference.<BR>- "Nice weather." for a sentiment analysis.<BR>- PNG file for an image transformation.<BR>- ["a", "b"] for a text summarization.<BR>- [1, 2] for a sales forcast.<BR>- [0.9, 0.1] for mnist data. |
+|option |string| Option field. Must be json format. |
+
+The "option" field needs to be a json format. Any style is Ok but we have some reserved fields below.
+
+|Field |Type |Description |
+|:---|:---|:---|
+|suppress_log_input |bool |True: NOT print the input and output to the log message. <BR>False (default): Print the input and outpu to the log message. |
+|YOUR KEY |any |YOUR VALUE |
+
+### `PredictResult` type
+*M* is the number of classes. If your algorithm is a binary classifier, you set *M* to 1. If your algorithm is a multi-class classifier, you set *M* to the number of classes.
+
+|Field |Type |Description |
+|:---|:---|:---|
+|label<BR>(required) |One of below<BR> -string<BR> -bytes<BR> -string[*M*]<BR> -int[*M*]<BR> -double[*M*] |Result of inference.<BR> -"positive" for a sentiment analysis.<BR> -PNG file for an image transformation.<BR> -["a", "b"] for a multi-class classification.<BR> -[1, 2] for a multi-class classification.<BR> -[0.9, 0.1] for a multi-class classification. |
+|score<BR>(required) |One of below<BR> -double<BR> -double[*M*] |Score of result.<BR> -0.98 for a binary classification.<BR> -[0.9, 0.1] for a multi-class classification. |
+|option |string |Option field. Must be json format. |
+
+### `EvaluateResult` type
+`EvaluateResult` is the evaluation score. *N* is the number of evaluation data. *M* is the number of classes. If your algorithm is a binary classifier, you set *M* to 1. If your algorithm is a multi-class classifier, you set *M* to the number of classes.
+
+|Field |Type |Description |
+|:---|:---|:---|
+|num<BR>(required)|int |Number of evaluation data. |
+|accuracy<BR>(required) |double |Accuracy. |
+|precision<BR>(required) |double[*N*][*M*] |Precision. |
+|recall<BR>(required) |double[*N*][*M*] |Recall. |
+|fvalue<BR>(required) |double[*N*][*M*] |F1 value. |
```

Comment:
I just found precision, recall and fvalue are `double[*M*]`

Reply:
I have updated!